tests/app/v2/test_errors.py

```diff
@@ -168,7 +170,4 @@ def test_bad_method(app_for_test):
 
             assert response.status_code == 405
 
-            assert response.json == {
-                "result": "error",
-                "message": "The method is not allowed for the requested URL.",
-            }
+            assert json.loads(response.data) == {"message": "The method is not allowed for the requested URL.", "result": "error"}
```

Comment:
you can't compare the `response.json` here?

Reply:
For some reason this comes back as `text/html` now instead of json.  Here is some more context from my commit message:
> this test was expecting a json response for the default MethodNotAllowed exception, which now seems to be text/html

Another note: the rest of the tests in this file test blueprints made explicitly in the test, where this one ends up testing the default `MethodNotAllowed` of werkzeug, which seems a bit odd to me.  In any case, I dont think we would rely on the default `MethodNotAllowed` to drive anything programatically.